src/pattern-library/examples/select-next-basic.tsx

```diff
@@ -17,18 +17,18 @@ export default function App() {
   return (
     <div className="w-96 mx-auto">
       <label htmlFor={selectId}>Select a person</label>
-      <SelectNext
+      <Select
         value={value}
-        onChange={setSelected}
+        onChange={newValue => setSelected(newValue)}
```

Comment:
I used an inline callback here just to demonstrate type is properly inferred with `Select`, as this file is type-checked.